client/components/editor/index.vue

```diff
@@ -58,7 +60,6 @@ export default {
       this.togglePublishDiff(false);
     },
     selectElement(element) {
-      this.selectedElement = element;
```

Comment:
If all elements are not going to create clone of element within Edit component, SSE might override their local changes for edited element. Do we need some kind of Edit cache; with last change wins strategy to avoid weird behaviour? Please check all potential side effects and decide about desired behaviour. 

Side note: I noticed that saving element meta (element configurable inputs; reach out to @abasic  / @kronicker  if you need help configuring it) are not properly being updated upon SSE change triggered by another user until user refocuses. That relates to previous topic / all sections should have similar behaviour.

Reply:
@underscope it seems to me that the behavior might be weird for one of the users in either case, as long as we're not showing that the content is being edited by the other user.
Do you think this solution would behave ok/predictably if we'd show that? Avoiding caching logic without a UX trade-off looks like a nice win. 😄